scripts/comet_archiver

```diff
@@ -5,34 +5,33 @@ import logging
 
 from comet.archiver import Archiver
 
-DEFAULT_DUMP_PATH = "./"
-DEFAULT_SCRAPE_INTERVAL = 360  # seconds
+DEFAULT_FAILURE_WAIT_TIME = 500  # milliseconds
 DEFAULT_BROKER_HOST = "127.0.0.1"
 DEFAULT_BROKER_PORT = 12050
+DEFAULT_REDIS_HOST = "127.0.0.1"
+DEFAULT_REDIS_PORT = 6379
 LOG_FORMAT = "[%(asctime)s] %(name)s: %(message)s"
 
 logging.basicConfig(format=LOG_FORMAT)
 
 
 def start(args):
-    comet = Archiver(args.dump_path, args.scrape_interval, args.broker_host, args.broker_port)
+    comet = Archiver(**vars(args))
     comet.run()
 
 
 parser = argparse.ArgumentParser(description="This is comet's archiver.")
 parser.add_argument(
-        "-d", "--dump-path",
-        help="set dump path (default: '{}')".format(DEFAULT_DUMP_PATH),
-        default=DEFAULT_DUMP_PATH
-        )
-parser.add_argument(
-        "-i", "--scrape-interval",
-        help="set scrape interval in seconds (default: '{}')".format(DEFAULT_SCRAPE_INTERVAL),
-        default=DEFAULT_SCRAPE_INTERVAL,
+        "-t", "--failure-wait-time",
```

Comment:
parsed_args will convert this into a variable named `failure_wait_time`?

Reply:
yes